src/test/Makefile

```diff
@@ -390,12 +401,14 @@ memcheck-summary-leaks:
 	grep "in use at exit" */memcheck*.log | grep -v " 0 bytes in 0 blocks" || true
 
 check:
+	@echo "Blacklisted tests: $(BLACKLIST_TESTS)"
```

Comment:
please don't print this line when BLACKLIST_TESTS is empty

Reply:
done